aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/AwsPresigningDecorator.kt

```diff
@@ -322,7 +322,16 @@ class AwsPresignedFluentBuilderMethod(
 
             let runtime_plugins = #{Operation}::operation_runtime_plugins(
                 self.handle.runtime_plugins.clone(),
-                self.config_override
+                self.config_override.map(|config_override| {
+                    crate::config::ConfigOverrideRuntimePlugin {
```

Comment:
Should `Operation::operation_runtime_plugins` do this mapping? It looks like its identical in both places it is done.

Reply:
165a013 now avoids a map operation at call sites and push that responsibility to the function body of `operation_runtime_plugins` which already has a check `if let Some(config_override) = config_override`.